handler/src/main/java/io/netty/handler/ssl/OpenSsl.java

```diff
@@ -313,12 +320,46 @@
                             SSL.freePrivateKey(key);
                         }
                     }
+
+                    String groups = SystemPropertyUtil.get("jdk.tls.namedGroups", null);
+                    if (groups != null) {
+                        String[] nGroups = groups.split(",");
+                        Set<String> supportedNamedGroups = new LinkedHashSet<String>(nGroups.length);
+                        Set<String> unsupportedNamedGroups = new LinkedHashSet<String>();
+                        for (String namedGroup : nGroups) {
+                            if (SSLContext.setCurvesList(sslCtx, namedGroup)) {
```

Comment:
This is just to test if the group members are supported? Or is this configuration meant to influence all ssl sessions?

Reply:
just for testing what is supported